File: tests/test_src/aplusb.pl
sub solveMeFirst{
    ($x,$y) = @_;
    # Hint: Type $res = $x + $y; below
    $res = $x + $y;
    return $res ;
}


$choice1 = <STDIN> ;
$choice2 = <STDIN> ;
$res = solveMeFirst($choice1 , $choice2);
print "$res" ;

File: tests/test_src/aplusb.php
<?php
function solveMeFirst($a,$b){
  // Hint: Type return $a + $b; below
  return $a + $b;
}
$handle = fopen ("php://stdin","r");
$_a = fgets($handle);
$_b = fgets($handle);
$sum = solveMeFirst((int)$_a,(int)$_b);
print ($sum);
fclose($handle);
?>
